### DISCUSSION TITLE
Dokan will not work, have to use WinFSP

### DISCUSSION THREAD

**Discussion by Glasairmell** [kubilus1/autoortho#412] [General]
After a lot of trouble shooting I found Dokan was not functioning for me on 6.0-1 or 5.1-3 for that matter.  Worked on 4.4.  I uninstalled it and used WinFSP.  This fixed my problems.

Any ideas why Dokan would not do the job for Auto Ortho?  It is the recommended option.

Cheers!

**Comment by cameroro** [comment]
How do you turn Dokan on and off, Or switch to WinFSP. What is the "Save button for?"
